docs/packages/all.rst

```diff
@@ -1,112 +1,32 @@
-* `cvmatio <https://github.com/ruslo/hunter/wiki/pkg.cvmatio>`__
-* `cxxopts <https://github.com/ruslo/hunter/wiki/pkg.cxxopts>`__
-* `dest <https://github.com/ruslo/hunter/wiki/pkg.dest>`__
-* `dlib <https://github.com/ruslo/hunter/wiki/pkg.dlib>`__
-* `doctest <https://github.com/ruslo/hunter/wiki/pkg.doctest>`__
-* `eos <https://github.com/ruslo/hunter/wiki/pkg.eos>`__
-* `flatbuffers <https://github.com/ruslo/hunter/wiki/pkg.flatbuffers>`__
-* `fmt <https://github.com/ruslo/hunter/wiki/pkg.fmt>`__
-* `freetype <https://github.com/ruslo/hunter/wiki/pkg.freetype>`__
-* `gflags <https://github.com/ruslo/hunter/wiki/pkg.gflags>`__
-* `glm <https://github.com/ruslo/hunter/wiki/pkg.glm>`__
-* `glog <https://github.com/ruslo/hunter/wiki/pkg.glog>`__
-* `half <https://github.com/ruslo/hunter/wiki/pkg.half>`__
-* `hdf5 <https://github.com/ruslo/hunter/wiki/pkg.hdf5>`__
-* `imshow <https://github.com/ruslo/hunter/wiki/pkg.imshow>`__
-* `intsizeof <https://github.com/ruslo/hunter/wiki/pkg.intsizeof>`__
-* `ios_sim <https://github.com/ruslo/hunter/wiki/pkg.ios_sim>`__
-* `irrXML <https://github.com/ruslo/hunter/wiki/pkg.irrXML>`__
-* `libuv <https://github.com/ruslo/hunter/wiki/pkg.libuv>`__
-* `libyuv <https://github.com/ruslo/hunter/wiki/pkg.libyuv>`__
-* `log4cplus <https://github.com/ruslo/hunter/wiki/pkg.log4cplus>`__
-* `lzma <https://github.com/ruslo/hunter/wiki/pkg.lzma>`__
-* `minizip <https://github.com/ruslo/hunter/wiki/pkg.minizip>`__
-* `msgpack <https://github.com/ruslo/hunter/wiki/pkg.msgpack>`__
-* `nlohmann::json <https://github.com/ruslo/hunter/wiki/pkg.nlohmann-json>`__
-* `odb-mysql <https://github.com/ruslo/hunter/wiki/pkg.odb-mysql>`__
-* `odb-pgsql <https://github.com/ruslo/hunter/wiki/pkg.odb-pgsql>`__
-* `ogles_gpgpu <https://github.com/ruslo/hunter/wiki/pkg.ogles_gpgpu>`__
-* `openddlparser <https://github.com/ruslo/hunter/wiki/pkg.openddlparser>`__
-* `poly2tri <https://github.com/ruslo/hunter/wiki/pkg.poly2tri>`__
-* `polyclipping <https://github.com/ruslo/hunter/wiki/pkg.polyclipping>`__
-* `rabbitmq-c <https://github.com/ruslo/hunter/wiki/pkg.rabbitmq.c>`__
-* `sparsehash <https://github.com/ruslo/hunter/wiki/pkg.sparsehash>`__
-* `spdlog <https://github.com/ruslo/hunter/wiki/pkg.spdlog>`__
-* `sse2neon <https://github.com/ruslo/hunter/wiki/pkg.sse2neon>`__
-* `szip <https://github.com/ruslo/hunter/wiki/pkg.szip>`__
-* `TacoPie <https://github.com/ruslo/hunter/wiki/pkg.tacopie>`__
-* `thread-pool-cpp <https://github.com/ruslo/hunter/wiki/pkg.thread-pool-cpp>`__
-* `tinydir <https://github.com/ruslo/hunter/wiki/pkg.tinydir>`__
-* `websocketpp <https://github.com/ruslo/hunter/wiki/pkg.websocketpp>`__
-* `wxWidgets <https://github.com/ruslo/hunter/wiki/pkg.wxwidgets>`__
-* `xgboost <https://github.com/ruslo/hunter/wiki/pkg.xgboost>`__
-* `yaml-cpp <https://github.com/ruslo/hunter/wiki/pkg.yaml-cpp>`__
+.. toctree::
+   :maxdepth: 1
+   :glob:
+
+   pkg/*
+
+Packages without description
```

Comment:
I don't think this one needed.

Reply:
I think it make sense to add all templates to main list. They can be fixed one-by-one later without moving from one folder to another.